hathor/p2p/protocol.py

```diff
@@ -86,15 +86,10 @@ class WarningFlags(str, Enum):
     aborting: bool
     diff_timestamp: Optional[int]
     idle_timeout: int
-    protocol_version: ProtocolVersion
+    sync_version: Optional[SyncVersion]
```

Comment:
Ours or theirs?

Reply:
This is the version chosen to be used on this protocol. So, both?

Reply:
Should we add a note about it? Maybe `# version chosen to be used on this connection`?

Reply:
Thanks!